patches/server/1047-Add-UUID-attribute-modifier-API.patch

```diff
@@ -15,14 +39,14 @@ index 0029412b1242879deb898524001bb4cc7550fa78..42917d79e20163a44f41c77f8a3d5e24
 +    // Paper start
 +    @Override
 +    public AttributeModifier getModifier(java.util.UUID uuid) {
-+        Preconditions.checkArgument(uuid != null, "uuid");
++        Preconditions.checkArgument(uuid != null, "UUID cannot be null");
```

Comment:
```suggestion
+        Preconditions.checkNotNull(uuid, "uuid");
```
This would also suffice as a not null check

Reply:
Used `Preconditions#checkArgument` to keep in style with the other methods in the class. Also, not a big deal, but `Preconditions#checkNotNull` throws a different exception (`NullPointerException` vs `IllegalArgumentException`).

Reply:
I checked some other CB classes and it is indeed a mix between `Preconditions.checkArgument(x != null, "x");` and `Preconditions.checkNotNull(x, "x cannot be null");`, so yeah that makes sense to keep in style with the surrounding code

Reply:
Yeah, I think checkArgument is better because we want the illegal argument exception to be thrown.